### PR TITLE
Replaced RST with MD file and completed edits.

### PR DESCRIPTION
**Pull request by MCamp859**
Signed-off-by: MCamp859 <maryx.camp@intel.com>
@felidadae @iwankgb @klihub  Please review these changes.

### CODE REVIEW

File: prm/demo_scenarios/README.md [hunk 1]
```diff
@@ -46,23 +51,20 @@ Assumptions:
...
-
-4. Reconfigure OWCA (by setting `action_delay` to 20 seconds) and restart it to run in `collect` mode
-
-.. code-block:: sh
-
+    ```
     ansible -u $REMOTE_USER -b all -i $REMOTE_IP, -mreplace -a'path=/etc/owca/owca_config.yml after=detector regexp="detect" replace="collect"'
     ansible -u $REMOTE_USER -b all -i $REMOTE_IP, -mreplace -a'path=/etc/owca/owca_config.yml regexp="action_delay: .*" replace="action_delay: 20."'
     ansible -u $REMOTE_USER -b all -i $REMOTE_IP, -msystemd -a'name=owca state=restarted'
+    ```
```
Comment (MCamp859) [inline prm/demo_scenarios/README.md]: @felidadae  2 instances of "owca_config.yml" above -- should it be ".yaml" instead?

File: prm/demo_scenarios/README.md [hunk 2]
```diff
@@ -74,14 +76,16 @@ Assumptions:
...
-
-.. code-block:: sh
-
-    # to check status
-    ansible -u $REMOTE_USER -b all -i $REMOTE_IP, -a'cat /etc/owca/owca_config.yml'
-    ansible -u $REMOTE_USER -b all -i $REMOTE_IP, -a'systemctl status owca'
-    ansible -u $REMOTE_USER -b all -i $REMOTE_IP, -a'ls -la /var/lib/owca/'
+```
+# to check status
+ansible -u $REMOTE_USER -b all -i $REMOTE_IP, -a'cat /etc/owca/owca_config.yml'
```
Comment (MCamp859) [inline prm/demo_scenarios/README.md]: @felidadae  One more instance of ".yml", should it be ".yaml"?

File: prm/demo_scenarios/README.md [hunk 3]
```diff
@@ -1,40 +1,45 @@
...
 
+## Assumptions
 
+Before you run the demo scenarios, you must have completed the following:
 
-1. Export required environment variables:
+*   Set the working directory to the Intel® PRM project top-level directory.
+*   Copy the template file from `demo_scenarios/common/common.template.yaml`
+    to `demo_scenarios/common/common.yaml` and fill it in.
+*   Add the `owca.pex` file with the Intel® PRM module included to the nodes
```
Comment (iwankgb) [inline prm/demo_scenarios/README.md]:
```suggestion
*   Copy the `owca.pex` file with the Intel® PRM module included to the nodes
```

File: prm/demo_scenarios/README.md (hunk 1 above)
Comment (iwankgb) [reply prm/demo_scenarios/README.md]: @MCamp859 I think it is down to inconsistent file extensions in our project. Leave it as it is, please.

**Pull request merged**